packages/react-error-overlay/webpack.config.iframe.js

```diff
@@ -19,7 +19,24 @@ module.exports = {
     rules: [
       {
         test: /\.js$/,
-        include: path.resolve(__dirname, './src'),
+        include: [
+          path.resolve(__dirname, './src'),
+          path.dirname(
+            require.resolve('chalk', {
+              paths: path.dirname(require.resolve('@babel/code-frame')),
+            })
+          ),
+          path.dirname(
+            require.resolve(
+              'ansi-styles',
+              path.dirname(
+                require.resolve('chalk', {
+                  paths: path.dirname(require.resolve('@babel/code-frame')),
+                })
```

Comment:
All of this looks very fragile, why did we add it?

Reply:
Oh, okay. https://github.com/facebookincubator/create-react-app/pull/3522#issuecomment-348370742

sigh

Reply:
It should be pretty reliable -- it resolves `chalk` from the context of `@babel/code-frame` (so we find the correct module which needs compiling).

Same for `ansi-styles` and `chalk`.

Alternatively, we can just always compile those modules.